docs/docs/download-buttons.md

```diff
@@ -4,22 +4,49 @@ render_macros: true
 
 # Firefox Download Buttons {: #download-buttons }
 
+To ensure that visitors to Firefox download pages get served the correct build and installer type for their operating system, we rely on [User Agent](https://developer.mozilla.org/docs/Glossary/User_agent) information to figure out what's needed. We primarily use the [Client Hints API](https://developer.mozilla.org/docs/Web/HTTP/Guides/Client_hints) to query this information, falling back to [navigator.UserAgent](https://developer.mozilla.org/docs/Web/API/Navigator/userAgent) for other web browsers.
+
```

Comment:
> This class is then used as a styling hook for displaying platform specific content

So the page contains all the logic, but we conditionally only show the relevant stuff? If I'm right there, how do we handle no-JS situations?

Reply:
There is a no-js selector / fallback - will add 👍